src/core_modules/capture-core/reducers/descriptions/activePage.reducerDescription.js

```diff
@@ -0,0 +1,30 @@
+// @flow
```

Comment:
Interesting. I'm not sure we want to generalize this to activePage. Or maybe we could for the isLoading and the error part.

Reply:
yeah that is another good point and I wanted to discuss it with you. In the next PR the stage of the `activePage` is more mature. Do you mind for the sake of clarity to discuss it there?